modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About external IP addresses
+
+You can allow external traffic into the cluster by defining an *external IP* configuration.
+
+The configuration for external IP address blocks is defined by a Network custom resource (CR) named `cluster`.
+The Network custom resource (CR) is defined by the `Network.config.openshift.io` CustomResourceDefinition.
+
+The object is created during cluster installation by the Cluster Version Operator (CVO).
+
+// If it is a cloud deployment, what happens?
```

Comment:
to answer your question, you can set up external IPs on cloud deployments, but it's not useful.